README.md

```diff
@@ -0,0 +1,34 @@
+# Crypt-Roller
+An expremential trial for exploring building a ZkRollup L2 using Rust as a programming lanageuage, and ZoKrates as a ZKP circuit lanaguage. 
+
+## Circuits
+### Installations
+
+- Installing ZoKrates compiler, checkout [Getting Started Steps on ZoKrates Docs](https://zokrates.github.io/gettingstarted.html)
+
+### Development
+- Compile Zokrates circuits.
+```bash
+make compile
+```
+The compilation output will be shown inside `circuits/out` directory.
+
+- Setup Zokrates circuits proving key. 
+```bash
+make setup
+```
+
+- Generating Zokrates verifier Solidity contract. 
+```bash
+make verifier
+```
+
+- Clean circuits compile output.
+```bash
+make clean
+```
+
+- To run all commands together `compile >> setup >> verifier` run:
+```bash
+make
+```
```

Comment:
I'd just leave the final part. That's all you need to build it and the rest can be easily checked in the `Makefile`.

If you want to have it more documented it, it would be more useful to say what the process actually does (i.e. what it compiles, into what, and what that's needed for). But we could just as well just point at the relevant point in Zokrates docs. Things will be in flux for a while so it's better if the docs are simple and do not require updating all the time :) Otherwise they will be constantly out of date.